internal/mesh/internal/controllers/sidecarproxy/builder/local_app_test.go

```diff
@@ -704,6 +704,392 @@ func TestBuildTrafficPermissions(t *testing.T) {
 					},
 				},
 			},
+			"destination rules l7, exclude rules for one port": {
+				defaultAllow: true,
+				workloadPorts: map[string]*pbcatalog.WorkloadPort{
+					"p1": {
+						Protocol: pbcatalog.Protocol_PROTOCOL_HTTP,
+					},
+					"p2": {
+						Protocol: pbcatalog.Protocol_PROTOCOL_HTTP,
+					},
+				},
+				ctp: &pbauth.ComputedTrafficPermissions{
+					AllowPermissions: []*pbauth.Permission{
+						{
+							Sources: []*pbauth.Source{
+								{
+									IdentityName: "foo",
+									Partition:    tenancy.Partition,
+									Namespace:    tenancy.Namespace,
+								},
+							},
+							DestinationRules: []*pbauth.DestinationRule{
+								{
+									PathPrefix: "/",
+									Methods:    []string{"get", "post"},
+									Exclude: []*pbauth.ExcludePermissionRule{
+										{
+											PathExact: "/secret",
+											Headers:   []*pbauth.DestinationRuleHeader{{Name: "restricted"}},
+											PortNames: []string{"p2"},
```

Comment:
The way exclude ports works as expected, but feels confusing. I wonder if we should just remove port names from excludes 🤔